Mathlib/NumberTheory/Padics/PadicIntegers.lean

```diff
@@ -306,8 +287,8 @@ theorem exists_pow_neg_lt {ε : ℝ} (hε : 0 < ε) : ∃ k : ℕ, (p : ℝ) ^ (
     norm_cast
     apply le_of_lt
     convert Nat.lt_pow_self _ using 1
-    exact hp.1.one_lt
-  · exact mod_cast hp.1.pos
+    exact (Fact.out : p.Prime).one_lt
```

Comment:
Why this change?

Reply:
It avoided having to refer to the `Fact` instance by name, but I've reverted that